openedx/features/course_experience/static/course_experience/js/spec/LatestUpdate_spec.js

```diff
@@ -0,0 +1,31 @@
+/* globals $, loadFixtures */
+
+import 'jquery.cookie';
+import { LatestUpdate } from '../LatestUpdate';
+
+
+describe('LatestUpdate tests', () => {
+  function createLatestUpdate() {
+    new LatestUpdate({ messageContainer: '.update-message', dismissButton: '.dismiss-message button' }); // eslint-disable-line no-new
+  }
+  describe('Test cookies', () => {
+    beforeEach(() => {
+      $.cookie('update-message', '');
+      loadFixtures('course_experience/fixtures/latest-update-fragment.html');
+    });
+
+    it('Test dismiss button', () => {
+      expect($.cookie('update-message')).toBe('');
+      createLatestUpdate();
+      expect($('.update-message').attr('style')).toBe(undefined);
+      $('.dismiss-message button').click();
+      expect($('.update-message').attr('style')).toBe('display: none;');
+    });
```

Comment:
Sorry if this is a WIP.  The point should be to start and end with an assertion, where the more important is the ending one:
```
expect($.cookie('update-message')).toBe('hide');
```
This could also assert on the expiration of the cookie if that is simple.